src/@lekoarts/gatsby-theme-minimal-blog/components/seo.tsx

```diff
@@ -19,27 +21,29 @@ const Seo = ({
   children = null,
   canonicalUrl = ``,
 }: SEOProps) => {
-  const site = useSiteMetadata()
-
+  const { language } = useI18next();
+  const site = useSiteMetadata();
+  const { t } = useTranslation();
   const {
-    siteTitle,
-    siteTitleAlt: defaultTitle,
+    // siteTitle, we can pass the i18n id here as well.
+    // siteTitleAlt: defaultTitle,
     siteUrl,
     siteDescription: defaultDescription,
     siteImage: defaultImage,
     author,
     siteLanguage,
-  } = site
+  } = site;
 
   const seo = {
-    title: title ? `${title} | ${siteTitle}` : defaultTitle,
+    title: title ? `${title} | ${t('site_title')}` : t('default_site_title'),
     description: description || defaultDescription,
```

Comment:
Could you please update site description as well?
![Screenshot by Dropbox Capture](https://github.com/kaleemullahdev/romanrud-blog/assets/12322543/cda00f43-7c90-402c-9b8b-28c8d9d6b4a8)

Reply:
![Screenshot by Dropbox Capture](https://github.com/kaleemullahdev/romanrud-blog/assets/12322543/63470691-ebc2-4ae9-bfb7-adf4a3f9e39c)